apps/codeforafrica/src/payload/utils/formatDraftUrl.js

```diff
@@ -0,0 +1,16 @@
+function formatDraftUrl(collection, doc, { locale }) {
+  const pagePath = `/${collection}/${doc.slug}`;
```

Comment:
Didn't we have a function to process this? For pages with parent, e.g. resources/datasets, the above code wouldn't work.